app.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+
+console.log('-----First and Pike Begins----');
+var firstAndPike = {
+  name: 'First and Pike',
+  minCustomer: 23,
+  maxCustomer: 65,
+  avgCookies: 6.3,
+  avgHours: [],
+  cookiesPerHour: function() {
+    for (var hours = 0; hours < 15; hours++) {
+      console.log('Inside loop ' + hours);
```

Comment:
Can you think of a way to make your console log statements more informative for you as a developer?